roles/obol_charon/tasks/main.yml

```diff
@@ -27,8 +27,19 @@
         secret_destination_path: "{{ obol_charon_private_key_file_path }}"
         secret_type: key
 
-# Get cluster lock file 
-# And also the validator_keys files from vault ?
+    - name: Fetch cluster lock file and save
+      ansible.builtin.include_tasks: tasks/get_and_save_vault_secret.yml
+      vars:
+        vault_secret_source_path: "{{ project }}/{{ org }}/{{ type }}/{{ obol_charon_lock_file_name }}"
+        secret_destination_path: "{{ obol_charon_lock_file_path }}"
+        secret_type: json
+
+    - name: Fetch validator_keys and save
+      ansible.builtin.include_tasks: tasks/get_and_save_vault_secret.yml
+      vars:
+        vault_secret_source_dir: "{{ project }}/{{ org }}/{{ type }}/validator_keys"
```

Comment:
lets define validator_keys_dir_name in vars main and use in here and in default main too